integration_tests/CMakeLists.txt

```diff
@@ -239,7 +248,8 @@ RUN(NAME test_for_loop       LABELS cpython llvm c)
 RUN(NAME modules_01          LABELS cpython llvm wasm wasm_x86)
 RUN(NAME modules_02          LABELS cpython llvm wasm wasm_x86)
 RUN(NAME test_import_01      LABELS cpython llvm)
-RUN(NAME test_import_02      LABELS cpython llvm)
+RUN(NAME test_import_02      IMPORT_PATH integration_tests
+        LABELS cpython llvm)
```

Comment:
Isn't this change equivalent to what there was before? I think it is relative to the current directory, which I am not sure if it is the directory with the sources, and the `integration_tests` is a non-existent directory then. Unless the working directory is the root LPython git repository's directory, which it is not if I do it by hand. So the directory doesn't exist, and LPython just finds the same modules as before.

Reply:
Correct. I just wanted to make sure that things work if we pass import paths separately. For example `main` fails if I pass `integration_tests` as import path (see example below). You can see even though behaviour of both commands should have been the same but it isn't. So I thought to test `IMPORT_PATH` as well.

```bash
(lp) 9:09:50:~/lpython_project/lpython % lpython -I integration_tests integration_tests/test_import_02.py
semantic error: Could not find the module 'test_import.import_module_01.py'. If an import path is available, please use the `-I` option to specify it
 --> integration_tests/test_import_02.py:1:1
  |
1 | from test_import.import_module_01 import multiply, e
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 


Note: if any of the above error or warning messages are not clear or are lacking
context please report it to us (we consider that a bug that needs to be fixed).
(lp) 9:10:30:~/lpython_project/lpython % lpython integration_tests/test_import_02.py 
2.71828182845904509e+00
1.45136923488338110e+00
30
200
```

Reply:
ok

Reply:
I guess we should make a directory outside integration_tests something called, `test_modules` and define our modules to be tested inside it. We pass the import path of `test_modules` in the command to check if it works.

Reply:
Yes.